awebox/mdl/aero/indicators.py

```diff
@@ -80,138 +87,149 @@ def get_performance_outputs(options, atmos, wind, variables, outputs, parameters
     outputs['performance']['p_current'] = current_power
 
     epsilon = 1.0e-8
-
     p_loyd_total = outputs['performance']['p_loyd_total']
     outputs['performance']['loyd_factor'] = current_power / (p_loyd_total + epsilon)
 
     outputs['performance']['power_density'] = current_power / len(kite_nodes) / parameters['theta0','geometry','s_ref']
 
     return outputs
 
-def collect_kite_aerodynamics_outputs(options, atmos, wind, ua, ua_norm, q_eff, aero_coefficients, f_aero, f_lift, f_drag, f_side, m_aero, ehat_chord, ehat_span, r, q, n, outputs, parameters):
+
+def collect_kite_aerodynamics_outputs(options, architecture, atmos, wind, variables, parameters, intermediates, outputs):
 
     if 'aerodynamics' not in list(outputs.keys()):
         outputs['aerodynamics'] = {}
 
-    for name in set(aero_coefficients.keys()):
-        outputs['aerodynamics'][name + str(n)] = aero_coefficients[name]
+    # unpack
+    kite = intermediates['kite']
+    air_velocity = intermediates['air_velocity']
+    aero_coefficients = intermediates['aero_coefficients']
+    f_aero_earth = intermediates['f_aero_earth']
+    f_aero_body = intermediates['f_aero_body']
+    f_aero_control = intermediates['f_aero_control']
+    f_aero_wind = intermediates['f_aero_wind']
+    f_lift_earth = intermediates['f_lift_earth']
+    f_drag_earth = intermediates['f_drag_earth']
+    f_side_earth = intermediates['f_side_earth']
+    m_aero_body = intermediates['m_aero_body']
+    kite_dcm = intermediates['kite_dcm']
+    q = intermediates['q']
 
-    outputs['aerodynamics']['v_app' + str(n)] = ua
-    outputs['aerodynamics']['airspeed' + str(n)] = ua_norm
-    outputs['aerodynamics']['u_infty' + str(n)] = wind.get_velocity(q[2])
+    verification_test = options['aero']['vortex']['verification_test']
+    if verification_test:
+        f_lift_earth, air_velocity = get_vortex_verification_f_lift_earth(intermediates['dq'])
 
-    rho = atmos.get_density(q[2])
-    outputs['aerodynamics']['air_density' + str(n)] = rho
-    outputs['aerodynamics']['dyn_pressure' + str(n)] = 0.5 * rho * cas.mtimes(ua.T, ua)
+    for name in set(intermediates['aero_coefficients'].keys()):
+        outputs['aerodynamics'][name + str(kite)] = intermediates['aero_coefficients'][name]
 
-    outputs['aerodynamics']['f_aero' + str(n)] = f_aero
-    outputs['aerodynamics']['f_lift' + str(n)] = f_lift
-    outputs['aerodynamics']['f_drag' + str(n)] = f_drag
-    outputs['aerodynamics']['f_side' + str(n)] = f_side
+    outputs['aerodynamics']['air_velocity' + str(kite)] = air_velocity
```

Comment:
this is me attempting to clear up the confusion, between ground_velocity, and air_velocity, since that was being mis-used in the initialization... it is the effective wind speed.

Reply:
ok thanks!